A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml

```diff
@@ -1190,6 +1190,24 @@
 	</UseTemplate>
 </Template>
 
+<Template Name="ASOBO_AIRLINER_GPWS_ALIGN_Template">
```

Comment:
If this is a new custom template, we've started using A32NX_ in the name instead of ASOBO_